clarifai.go

```diff
@@ -175,9 +175,9 @@ func getRequestBody(input string) (*bytes.Buffer, error) {
 }
 
 // Get image tag array from Clarifai's API response
-func getImageTagsArray(jsonStr string) ([]ClarifyTag, error) {
+func getImageTagsArray(jsonStr string) ([]ImageTag, error) {
```

Comment:
Somehow answered this from the above comment.
I think this rule doesn't apply for functions. All functions should be globally accessible across the same package